web/routes.go

```diff
@@ -39,6 +40,12 @@ var routes = []route{
 		"/health",
 		healthHandler,
 	},
+	route{
+		"job_status",
```

Comment:
I think either convention is good. Let's go with the convention we were already using for consistency.